File: Python/Lab1/Variant 14.py
from math import *
a = float(input("Введите значение a: "))
b = float(input("Введите значение b: "))
x = float(input("Введите значение x: "))
y = pow(sin((x**2+a)**2),3)-sqrt(x/b)
print("\nЗначение y =", y)

File: Pascal/Lab1/Variant14.pas
Program lab1;
Uses Crt;
Var a,b,x,Formula: real;
Begin
  Write('Введите первое число: '); Readln(a);
  Write('Введите второе число: '); Readln(b);
  Write('Введите третье число: '); Readln(x);
  ClrScr;
  Formula:=power(sin(sqr(sqr(x)+a)),3)-sqrt(x/b);
  Writeln('Вариант 14.  ',Formula);
End.